src/views/orb-map-view.ts

```diff
@@ -89,6 +90,7 @@ export class OrbMapView<N extends INodeBase, E extends IEdgeBase> implements IOr
           this.render();
         }
       },
+      listeners: [this._update],
```

Comment:
It would be great to add a comment why this line and not `this`. Same for map view.

Reply:
I've added this comment in `observer.utils.ts` so we don't have to add it everywhere. Is it fine, or should I add it to the view files as well?